uservice_ccutter/plugins/load_plugin.py

```diff
@@ -0,0 +1,30 @@
+"""Load a plugin module by name"""
+import importlib
+import sys
+from apikit import BackendError
+try:
+    from importlib import ModuleNotFoundError
+except ImportError:
+    ModuleNotFoundError = ImportError
+
+
+def load_plugin(plugin_name):
+    """Load a named plugin.  Substitute "-" with "_" first.  If the module
+    is already loaded, return it; if not, load it and then return the
+    loaded module."""
+    ctt = "." + plugin_name.lower().replace("-", "_")
+    ctr = ".".join(__name__.split(".")[:-1])
+    modname = ctr + ctt
+    mod = None
```

Comment:
I can leave out the mod=None line (Python is not Go), but I would need to add another return if I were to move the return inside the try.

There are two tries there.  If the module is already loaded, return it.  If not, try to load it, and then return it.